instructor/ui/sketch_tool.js

```diff
@@ -18,29 +22,33 @@ function draw_graph_background(x_upper_bound, tick_values){
         draw_line(0, y, 600, y, '#eee');
     }
     /* x axis */
-    draw_line(X_ORIGIN, Y_ORIGIN, 560, Y_ORIGIN);
+    draw_line(X_ORIGIN, Y_ORIGIN, X_AXIS_LAST_VALUE, Y_ORIGIN);
     /* y axis */
-    draw_line(X_ORIGIN, 20, X_ORIGIN, Y_ORIGIN);
-    /* arrows */
-    draw_line(X_ORIGIN, 20, 30, 30);
-    draw_line(X_ORIGIN, 20, 50, 30);
-
-    draw_line(550, 250, 560, Y_ORIGIN);
-    draw_line(550, 270, 560, Y_ORIGIN);
+    draw_line(X_ORIGIN, Y_ORIGIN - Y_AXIS_LENGTH_PX, X_ORIGIN, Y_ORIGIN);
+    /* y axis arrows */
+    draw_line(X_ORIGIN, Y_ORIGIN - Y_AXIS_LENGTH_PX, X_ORIGIN - 10, Y_AXIS_LAST_VALUE + 10);
+    draw_line(X_ORIGIN, Y_ORIGIN - Y_AXIS_LENGTH_PX, X_ORIGIN + 10, Y_AXIS_LAST_VALUE + 10);
+    /* x axis arrows */
+    draw_line(X_AXIS_LAST_VALUE - 10, Y_ORIGIN - 10, X_AXIS_LAST_VALUE, Y_ORIGIN);
+    draw_line(X_AXIS_LAST_VALUE - 10, Y_ORIGIN + 10, X_AXIS_LAST_VALUE, Y_ORIGIN);
 
     /* Calculating axis labels */
     tick_values=tick_values.split(",").map(function(x){
         return parseInt(x)
     });
     x_upper_bound = parseInt(x_upper_bound);
-    _.each(tick_values, function(value, index){
-        var x_coor = X_AXIS_LENGTH_PX * value / x_upper_bound + X_ORIGIN;
-        var y_coor = Y_AXIS_LARGEST_PX + 13;
-        var text_obj = new paper.PointText(new paper.Point(x_coor, y_coor));
-        text_obj.content = value;
-        // need to subtract half of the width from x
-        text_obj.position.x -= text_obj.bounds.width / 2;
+    var x_coor, y_coor;
+    _.each(tick_values, function(value){
+        x_coor = X_AXIS_LENGTH_PX * value / x_upper_bound + X_ORIGIN;
+        y_coor = Y_ORIGIN + 13;
+        printText(x_coor, y_coor, value);
+    });
+    _.each([20, 40, 60, 80], function(value){
```

Comment:
Are the y axis tick marks fixed at 20, 40, 60, 80? If they are, we should add a comment here.